hack/helm/release/readme.md

```diff
@@ -0,0 +1,91 @@
+# ECK Helm Chart Releaser
```

Comment:
I'm a bit concerned about maintaining this readme with this high level of details. The tool already has a `--help` flag that covers a lot of information. We could stay more general.